packaging/deb-v2/debian/control

```diff
@@ -0,0 +1,12 @@
+Source: mackerel-agent-plugins
+Maintainer: Mackerel Developers <mackerel-developers@hatena.ne.jp>
+Section: misc
+Priority: optional
+Build-Depends: debhelper (>= 9)
+Standards-Version: 3.9.2
```

Comment:
https://travis-ci.org/mackerelio/mackerel-agent-plugins/builds/236927983#L1490

The travis box says current is 3.9.3 so, I'll bump to 3.9.3.